PB install with nodejs 5.1.1

I have a problem to exexute command :
```
npm install gulp-rewrite-css
```
the error :
```
npm WARN prefer global coffee-script@1.8.0 should be installed with -g

> gulp-rewrite-css@0.0.4 postinstall E:\Projets\xxxxx\node_modules\gulp-rewrite-css
> ./node_modules/coffee-script/bin/coffee -o lib/ src/

'.' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
npm WARN EPACKAGEJSON xxxxx No description
npm WARN EPACKAGEJSON xxxxx No repository field.
npm WARN EPACKAGEJSON xxxxx No license field.
npm ERR! Windows_NT 10.0.10240
npm ERR! argv "H:\\My Programs\\nodejs\\node.exe" "H:\\My Programs\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install
" "gulp-rewrite-css"
npm ERR! node v5.1.1
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE

npm ERR! gulp-rewrite-css@0.0.4 postinstall: `./node_modules/coffee-script/bin/coffee -o lib/ src/`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gulp-rewrite-css@0.0.4 postinstall script './node_modules/coffee-script/bin/coffee -o lib/ src/'.

npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the gulp-rewrite-css package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/coffee-script/bin/coffee -o lib/ src/
npm ERR! You can get their info via:
npm ERR!     npm owner ls gulp-rewrite-css
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
```
I'am on windows 10 pro with nodejs 5.1.1

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
+  - "stable"
+  - "4.2"
   - "0.12"
   - "0.11"
   - "0.10"

--- a/package.json
+++ b/package.json
@@ -19,11 +19,11 @@
     "node": ">= 0.10"
   },
   "scripts": {
-    "prepublish": "./node_modules/.bin/coffee -o lib/ src/",
+    "prepublish": "coffee -o lib src",
     "test": "mocha --compilers coffee:coffee-script/register -R spec",
     "watch": "mocha --compilers coffee:coffee-script/register -R min --watch",
-    "coverage": "./node_modules/.bin/coffee -o src/ src/ && ./node_modules/.bin/coffee -o test/ test/ && ./node_modules/.bin/istanbul cover node_modules/.bin/_mocha --report html -- -R spec -t 3000 -s 2000",
-    "coveralls": "./node_modules/.bin/coffee -o src/ src/ && ./node_modules/.bin/coffee -o test/ test/ && ./node_modules/.bin/istanbul cover _mocha --report lcovonly -- -R spec -t 3000 -s 2000 && cat ./coverage/lcov.info | ./node_modules/.bin/coveralls"
+    "coverage": "coffee -o src src && coffee -o test test && istanbul cover _mocha --report html -- -R spec -t 3000 -s 2000",
+    "coveralls": "coffee -o src src && coffee -o test test && istanbul cover _mocha --report lcovonly -- -R spec -t 3000 -s 2000 && cat ./coverage/lcov.info | coveralls"
   },
   "devDependencies": {
     "mocha": "~2.3.3",

